Adding dogpile.cache for caching skopeo_inspect (#142)
release-engineering/iib: README.md

  Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
  registry of the `from_index` container image when applicable. IIB will never directly modify this
  file though. This defaults to `~/.docker/config.json.template`.
*  `iib_dogpile_backend` - configuration for dogpile.cache backend. Default value is

>    `'dogpile.cache.memcached'`. In case you want to disable caching 

Reviewer: This looks like an incomplete sentence.

> *  `iib_dogpile_expiration_time` - Number of seconds after which cached item is expired.

Reviewer: ```suggestion
*  `iib_dogpile_expiration_time` - The number of seconds after which the cached item is expired.
```

>    `iib_dogpile_arguments` - Additional arguments for dogpile backed.

Reviewer: ```suggestion
   `iib_dogpile_arguments` - Additional arguments for dogpile backend.
```

  Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
  registry of the `from_index` container image when applicable. IIB will never directly modify this
  file though. This defaults to `~/.docker/config.json.template`.

> *  `iib_dogpile_backend` - configuration for dogpile.cache backend. Default value is

Reviewer 2: Could you please document `iib_dogpile_expiration_time` and `iib_dogpile_arguments` in the readme too?

Author: Sure, thanks

  Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
  registry of the `from_index` container image when applicable. IIB will never directly modify this
  file though. This defaults to `~/.docker/config.json.template`.

> *  `iib_dogpile_backend` - configuration for dogpile.cache backend. Default value is
>    `'dogpile.cache.null'`. In case you want to enable caching use `'dogpile.cache.memcached'`.

Reviewer: ```suggestion
*  `iib_dogpile_backend` - The configuration for the dogpile.cache backend. The default value is
   `'dogpile.cache.null'`. In case you want to enable caching, set this to `'dogpile.cache.memcached'`.
```

>    `'dogpile.cache.memcached'`. In case you want to disable caching use `'dogpile.cache.null'`.

Reviewer: The default here needs to be changed to the new default of 'dogpile.cache.null'`.

Author: Thank you, updated.

*  `iib_dogpile_expiration_time` - The number of seconds after which the cached item is expired.

>    `iib_dogpile_arguments` - Additional arguments for dogpile backend.

Reviewer: ```suggestion
   `iib_dogpile_arguments` - Additional arguments for the dogpile backend.
```

>    (e.g. for testing), you can set this variable to `'dogpile.cache.null'`.

Reviewer: ```suggestion
```

  Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
  registry of the `from_index` container image when applicable. IIB will never directly modify this
  file though. This defaults to `~/.docker/config.json.template`.

> *  `iib_dogpile_backend` -  The configuration for the dogpile.cache backend. The default value is

Reviewer: To be consistent with the rest of the document.
```suggestion
*  `iib_dogpile_backend` -  the configuration for the dogpile.cache backend. The default value is
```

> *  `iib_dogpile_backend` -  the configuration for the dogpile.cache backend. The default value is

Reviewer: ```suggestion
*  `iib_dogpile_backend` - the configuration for the dogpile.cache backend. The default value is
```

Author: Updated, thanks.

   `'dogpile.cache.null'`. In case you want to enable caching, set this to `'dogpile.cache.memcached'`.

> *  `iib_dogpile_expiration_time` - The number of seconds after which the cached item is expired.

Reviewer: To be consistent with the rest of the document.
```suggestion
*  `iib_dogpile_expiration_time` - the number of seconds after which the cached item is expired.
```

>    `iib_dogpile_arguments` - Additional arguments for the dogpile backend.

Reviewer: To be consistent with the rest of the document.
```suggestion
   `iib_dogpile_arguments` - additional arguments for the dogpile backend.
```
